spec/models/role_spec.rb

```diff
@@ -8,7 +8,7 @@
   describe "::ROLES" do
     it "contains the correct values" do
       expected_roles = %w[
-        admin banned chatroom_beta_tester codeland_admin comment_banned podcast_admin pro restricted_liquid_tag
+        admin suspended chatroom_beta_tester codeland_admin comment_suspended podcast_admin pro restricted_liquid_tag
```

Comment:
**nitpick (non-blocking):** These were alphabetically sorted before, but changing `banned` to `suspended` changes that.